fluss-client/src/test/java/com/alibaba/fluss/client/admin/FlussAdminITCase.java

```diff
@@ -562,6 +556,206 @@ void testGetServerNodes() throws Exception {
         assertThat(serverNodes).containsExactlyInAnyOrderElementsOf(expectedNodes);
     }
 
+    @Test
```

Comment:
Add test cases for `SHOW PARTITIONS table_name` to verify the support for `FlinkCatalog#listPartitions`.

Reply:
+1

Reply:
Done!